### DISCUSSION TITLE
Any specific date for a release?

### DISCUSSION THREAD

**Discussion by thereis** [diced/zipline#463] [Questions] [answered]
Recently I've pushed a new commit as a fix for the upcoming release which I really want to see it working... are there specific dates when you guys creates a new release?

**Comment by diced** [comment, marked as answer]
Since v4 is being developed currently, updates to the current v3 codebase are slow but do happen sometimes. I will make a new 3.7.x release when there are a bit more bug fixes to warrant a patch release. If you want to run the latest commit, you can just use the `trunk` tag on the docker image, or just `git pull`.
